File: Issue-Tracker.Server/Data/Issue.cs
public class Issue {
    
    public int IssueId { get; set; }
    public string? IssueName { get; set; }
    public string? Description { get; set; }
    public string? Department { get; set; }
    public Status? Status { get; set; }
    public DateTime DateWorking { get; set; }
    public bool Active { get; set; }
}

File: issue-tracker.client/src/app/models/Issue.ts
import { Priority } from "./enums/Priority";
import { Status } from "./enums/Status";

export interface Issue {
    issueId: number;
    issueName?: string;
    description?: string;
    department?: string;
    status: Status | undefined;
    priority: Priority | undefined;
    dateWorking: Date;
    active: boolean;
  }